modules/ensemblvep/meta.yml

```diff
@@ -47,6 +36,10 @@ input:
       type: file
       description: |
         path to VEP cache (optional)
+  - extra_files:
+      type: tuple
+      description: |
+        path to file(s) needed for plugins  (optional)
```

Comment:
single path to a directory containing, or list of paths?

Reply:
either

Reply:
I guess either could work